conf/messages.en

```diff
@@ -895,7 +895,7 @@ declaration.declarationHolders.add.another.empty = Select yes if you need to add
 declaration.declarationHolders.remove.empty = Select yes if you want to remove this authorisation
 declaration.declarationHolder.remove.title = Are you sure you want to remove this authorisation?
 declaration.declarationHolder.error.duplicate = You have already entered this EORI number for this authorisation
-declaration.declarationHolder.error.arrived = You need to select ‘Yes’ then enter an authorisation for the arrived declaration - this can be EXRR (no proof of this authorisation is required) or CSE if you have a CSE authorisation number.
+declaration.declarationHolder.error.arrived = You need to select 'Yes' and enter an authorisation for the arrived declaration
```

Comment:
The single quotes are not correct, isn't it?

Reply:
Fixed :+1: